### PR TITLE
[F2F-122]: Call BE /claimedIdentity and return browser redirect to IPV Core

### PR DESCRIPTION
**Pull request by richa-misra**
### What changed
Adding call to /claimedIdentity API when user confirm their details on check answers screen

### Issue tracking
- [F2F-122](https://govukverify.atlassian.net/browse/F2F-122)

### Environment variables or secrets

- [ ] Documented in the README
- [X] Added screenshots to show the implementation is working

https://user-images.githubusercontent.com/26764987/219049074-edca3255-b32d-411a-9736-a6e5d2b7149b.mov

Screenshot 2023-02-15 at 14 08 46

- [ ] Ran cfn-lint on any SAM templates

### Testing
Tested by mocking the BE endpoint and adding browser based tests to check the flow
Also tested by deploying /claimedIdentity changes in [this PR](https://github.com/alphagov/di-ipv-cri-cic-api/pull/17) on `di-ipv-cri-dev `account and testing against it via the FE screen

[F2F-122]: https://govukverify.atlassian.net/browse/F2F-122?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/app/cic/controllers/checkDetails.js [hunk 1]
```diff
@@ -57,20 +57,57 @@ class CheckDetailsController extends DateController {
...
+    } catch (err) {
+      callback(err);
+    }
+
+  }
+
+  async saveCicData(axios, cicData, req) {
+
+    const headers = {
+      "session-id": req.session.tokenId,
```
Comment (CoralSmith) [inline src/app/cic/controllers/checkDetails.js:102]: We should pick one of these and make that the header that the API uses. These two were both present in the common CRI interface for tech debt reasons. 

I recommend separating using the dash and just documenting that as our standard from now on in.

File: test/browser/step_definitions/details.js [hunk 2]
```diff
@@ -23,14 +23,6 @@ Then("they should be redirected to the landingPage", async function () {
 });
 
 
-Then("they should be redirected as a success", function () {
```
Comment (CoralSmith) [inline test/browser/step_definitions/details.js:26]: I can see that this Then is used in some of the feature files but its flagged as being removed. Is this a duplicate?

File: src/app/cic/controllers/checkDetails.js (hunk 1 above)
Comment (richa-misra) [reply src/app/cic/controllers/checkDetails.js:102]: I did try keeping just the dash one earlier but it doesn't seem to work well, will catch up tomorrow to demo it

File: test/browser/step_definitions/details.js (hunk 2 above)
Comment (richa-misra) [reply test/browser/step_definitions/details.js:26]: thats right its a duplicate step

**Review by CoralSmith — APPROVED** [review]

**Review by shenry99 — APPROVED** [review]

**Pull request merged**